README.md

```diff
@@ -2,27 +2,48 @@
 Repo for GSOC Project Described at https://wiki.fluidproject.org/display/fluid/Google+Summer+of+Code+2018+with+the+Fluid+Project#GoogleSummerofCode2018withtheFluidProject-Buildaserviceforadaptivecontentandlearningsupports
 
 ## Instructions to run the service locally
-**Step 1 - Clone the Repository**\
-**Step 2 - Install the required node modules**\
+### Step 1 - Clone the Repository
+### Step 2 - Install the required node modules
 Run
 ```
 npm install
 ```
-**Step 3 - Setting API keys for services that require it**\
```

Comment:
I believe this should be updated with instructions for adding the Yandex key.

Reply:
Sorry, my bad.
Will add instructions for Yandex.

Reply:
One thing I'd suggest (I commonly do this for projects that need a secrets-type config file) is to create a sample `.env` file called something like `.env.example` that shows the expected structure and key names, without actually containing real secrets. Something like:

```
OXFORD_APP_ID=[Oxford app id here]
OXFORD_APP_KEY=[Oxford app key here]
```

You can then refer to this in the documentation and commit it to version control - this can help in making sure your config file documentation is up to date.

Reply:
Great idea. Will surely do the same here.